api/app/services/simplified_rag_service.py

```
@@ -177,6 +178,65 @@ def _initialize_xai_llm(self):
             logger.info("Falling back to OpenAI model.")
             self._initialize_openai_llm()
 
+    def _retrieve_with_version_priority(self, query: str, score_threshold: float = 0.3) -> List[Document]:
```

Ruff (0.13.3)

Unused method argument: `score_threshold`

(ARG002)